CONTRIBUTING.md

```diff
@@ -1116,7 +1116,7 @@ Then, [Updating language files](#updating-language-files).
 
 The topic of authoring new plugins, whether you'd like to contribute
 it back or not has been well documented in the
-[So, You Want to Build a Superset Viz Plugin...](https://preset.io/blog/2020-07-02-hello-world/) blog post
+[Building Custom Viz Plugins in Superset v2 (Updated for Monorepo)](https://preset.io/blog/building-custom-viz-plugins-in-superset-v2) blog post
```

Comment:
We now have official docs for this - should we link to the docs or maybe both? https://superset.apache.org/docs/contributing/creating-viz-plugins